cocos/3d/framework/mesh-renderer.ts

```diff
@@ -398,7 +398,8 @@ export class MeshRenderer extends ModelRenderer {
 
     public onGeometryChanged () {
         if (this._model && this._mesh) {
-            this._model.createBoundingShape(this._mesh.struct.minPosition, this._mesh.struct.maxPosition);
+            const meshStruct = this._mesh.struct;
```

Comment:
Is the purpose here to reduce temporary objects?

Reply:
Yes, also a bit of tweak for performance since less property visit.